_posts/2020-07-29-galaxy-20-05.md

```diff
@@ -0,0 +1,43 @@
+---
+site: freiburg
+tags: [release]
+title: UseGalaxy.eu update to 20.05
+supporters:
+ - denbi
+ - elixir
```

Comment:
btw. you can add yourself as author here...

Reply:
no need :)